pallets/liquid-staking-two/src/types.rs

```diff
@@ -0,0 +1,110 @@
+use codec::{Decode, Encode};
+use sp_runtime::{FixedPointNumber, RuntimeDebug};
+
+//TODO(wangyafei): use associated type.
+use primitives::{Balance, Rate};
+
+#[derive(Copy, Clone, Eq, PartialEq, Encode, Decode, RuntimeDebug)]
+pub enum StakingOperationType {
+    Bond,
+    Unbond,
+    Rebond,
+    Matching,
+    TransferToRelaychain,
```

Comment:
we dont need `TransferToRelayChain` because xTokens already offer it

Reply:
KSM can only be transfered from this pallet, not xTokens pallet,